Mayhemfile

```diff
@@ -0,0 +1,19 @@
+# Namespaced project name that the target belongs to
```

Comment:
Multiple commands are not supported; please either use only one command or split into multiple Mayhemfiles. Also, omit image name and use generic project name.